includes/overlords/contribs.php

```diff
@@ -175,6 +207,43 @@ public static function display_contribs($mode, $id, $branch = false, $sort = fal
 			break;
 
 			case 'category' :
+				// Get a list of hidden categories, so that if a category has been set to invisible we won't show
+				// the contribs in the parent category listing
+				$all_subcategory_ids = array();
+
+				foreach ($hidden_categories_ids as $hidden_category_id)
+				{
+					$all_subcategory_ids[] = (int) $hidden_category_id;
+					$children_ids = array_keys($cache->get_category_children($hidden_category_id));
+
+					foreach ($children_ids as $child_id)
```

Comment:
So this will only check one level of children, and presumably miss further potential nested layers of children?

Reply:
I've rewritten this and the other one